src/style.css

```diff
@@ -0,0 +1,109 @@
+html {
+  font-family: "Roboto", sans-serif;
+  margin: 0;
+}
+  
+* {
+  box-sizing: border-box;
+}
+
+.card {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  width: 200px;
+  padding: 29px 14px 15px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: auto;
+  margin-bottom: 33px;
+}
+
+.card__name {
+  margin-bottom: 6px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+}
+
+.card__code {
+  margin-bottom: 15px;
+  font-size: 11px;
+  line-height: 13px;
+}
+
+.card__rating {
+  margin-bottom: 20px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.stars {
+  display: flex;
+  align-items: center;
+}
+
+.stars__item {
+  margin-right: 4px;
+  height: 15px;
+  width: 15px;
+  background-image: url("images/star.svg");
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars :nth-child(-n+4) {
+  background-image: url("images/star-active.svg");
+}
+
+.card__respond {
+  font-size: 11px;
+  line-height: 13px;
+}
+
+.card__price {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 20px;
+}
+
+.price__text {
+  font-size: 11px;
+  line-height: 13px;
+}
+
+.price__sum {
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 19px;
+}
+
+.card__buy {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 40px;
+  background: #00acdc;
+  border-radius: 5px;
+  font-weight: bold;
+  font-size: 14px;
+  line-height: 16px;
+  text-transform: uppercase;
+  color: #fff;
+  text-decoration: none;
+}
+
+.card__buy:hover {
+  background: #ffff;
+  border: 1px solid #00acdc;
```

Comment:
move border in card_buy becouse when you hover on your link, text could change it's position